lib/screens/profile/add_friends.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:eventure/providers/user_provider.dart';
+import 'package:eventure/widgets/inputs/custom_input_line.dart';
+import 'package:eventure/widgets/widgets.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../models/user.dart';
+
+class AddFriendsScreen extends StatefulWidget {
+  const AddFriendsScreen({super.key});
+  @override
+  _AddFriendsScreenState createState() => _AddFriendsScreenState();
+}
+
+class _AddFriendsScreenState extends State<AddFriendsScreen> {
+  @override
+  void initState(){
+    super.initState();
+    context.read<UserProvider>().fetchUsersStartingWith();
+  }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text("Search users, add friends"),
+        ),
+        body: Column(
+          children: [
+            CustomInputLine(
+                label: "Enter other user's username",
+                required: false,
+                editable: true,
+                onChanged: (text) {
+                  context.read<UserProvider>().queryStartsWith=text;
+                  context.read<UserProvider>().fetchUsersStartingWith();
+                }),
+            Consumer<UserProvider>(builder: (context, userProvider, child) {
+              userProvider.fetchFriends();
+              List<AppUser> usersStartingWith = userProvider.usersStartingWith;
+              return Expanded(
+                  child: ListView.builder(
+                itemCount: usersStartingWith.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return ListTile(
```

Comment:
Wenn man auf ein ListTyle klickt wäre es auch cool, wenn man auf die Detail View des users weitergeleitet wird mit noch mehr Infos zum User. 
Evtl. muss dann noch das "user_profile" ein bisschen abgeändert werden, aber es sind ja eigentlich die gleichen Infos außer, dass man das Profil dann nicht bearbeiten kann.

Reply:
Habe gesehen dass es für die User Detail View noch ein Issue gibt. #30 
Also musst du das in diesem PR nicht machen